pages/getting-started/developers.md

```diff
@@ -140,6 +140,24 @@ You can also email us directly at [uswebdesignstandards@gsa.gov](mailto:uswebdes
 **For more information, visit:
 [https://pages.18f.gov/frontend/css-coding-styleguide/](https://pages.18f.gov/frontend/css-coding-styleguide/)**
 
+## JS customization
+
+The JavaScript for the standards is separated into components in the same manner as the visual interface which is all initialized with event handlers when the DOM is ready. These components are accessible as CommonJS modules that can be required in other JavaScript files which then must be fuilt for the browser. The components are currently not accessible in the global browser scope, but can be easily extended to be included by requiring `components` and setting it to a global scope:
```

Comment:
I wonder if it might be useful to just telegraph the fact that customizing USWDS's JS currently *requires* a tool like browserify or webpack?  E.g. a big bold paragraph at the beginning of this section that says:

> **Unfortunately, customizing the JavaScript for the standards currently requires NodeJS and a module bundler like Browserify or Webpack. We apologize for this inconvenience, and are working to resolve it in a future release of the Standards.**

Or something like that?  I'm concerned that folks who might not be super familiar with JS, e.g. who might not know exactly what "accessible in global browser scope" mean but who do know how to call functions and such, might be confused here. I mean, they might also not really know what node/babel/webpack are, but at least an apologetic tone could help convey the notion that leveraging our JS requires relatively advanced skills right now, especially compared to systems like Bootstrap and Foundation, and we're trying to lower the barrier to entry.